specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -0,0 +1,1143 @@
+      "get": {
+        "description": "List operations available for the Maps Resource Provider",
+        "operationId": "Accounts_ListOperations",
+        "x-ms-examples": {
+          "GetOperations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null,
+          "itemName": "value"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/MapsOperations"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/privateAtlas/{privateAtlasName}": {
+      "put": {
+        "description": "Create or update a Private Atlas resource. Private Atlas resource will enable the usage of Azure resources to build a custom set of mapping data. It requires an account to exist before it can be created.",
+        "operationId": "PrivateAtlasCollection_CreateOrUpdate",
+        "x-ms-examples": {
+          "CreateAccount": {
+            "$ref": "./examples/CreatePrivateAtlas.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateAtlasNameParameter"
+          },
+          {
+            "name": "PrivateAtlasCreateParameters",
+            "in": "body",
+            "description": "The new or updated parameters for the Private Atlas resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlasCreateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Private Atlas was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "201": {
+            "description": "The Private Atlas was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "202": {
+            "description": "The Private Atlas was accepted for creation",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
```

Comment:
you shouldn't need 202 here.  See [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources)